docs/mismatchfinder-api-spec.yml

```diff
@@ -215,9 +224,16 @@ components:
           type: string
           format: date
         uploader:
-          type: object
           $ref: '#/components/schemas/User'
 
+    FailedImportMeta:
+      allOf:
+      - $ref: '#/components/schemas/ImportMeta'
+      - type: object
+        properties:
+          error:
+            type: string
+
     ListOfImportMeta:
```

Comment:
this list is returned by `/imports`, and it can now contain instances of both `ImportMeta` and `FailedImportMeta`

Reply:
:+1: